### DISCUSSION TITLE
strom control

### DISCUSSION THREAD

**Discussion by srikanthyiza** [NatLabRockies/ROSCO#267] [Q&A]
Hello. I want know to know design of strom controller using PI in above the cut-out wind speed to avoid frequent shutdowns. someone in this group could you please help me.  Thanks inadvance

with regards
srikanth

**Comment by dzalkind** [comment]
A soft cut-out at high wind speeds is a recent addition to the develop branch.  An example can be found here: https://github.com/NREL/ROSCO/blob/develop/Examples/27_power_ref_control.py

More documentation can be found here, under power reference control: https://github.com/NREL/ROSCO/blob/develop/docs/source/api_change.rst
